docs/dcp2_system_design.rst

```diff
@@ -1199,6 +1222,72 @@ for a project with one non-stratified matrix.
    that file may have a more up-to-date list
 
 
+Describing CGMs as analysis files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The mechanism defined in `Describing CGMs as supplementary files`_ is
+deprecated. Instead of representing a contributor-generated matrix (CGM) as a
+stand-alone supplementary file, it is described more accurately as an
+``analysis_file`` entity that is an output of an ``analysis_process`` entity.
```

Comment:
Ingest are not using ‘analysis_process’ as the entity to output an ‘analysis_file’. We are using ‘process’. 

This is because ‘[analysis_process](https://github.com/HumanCellAtlas/metadata-schema/blob/master/json_schema/type/process/analysis/analysis_process.json)’ has a number of different fields which are required that we do not believe we will be able to capture from contributors on an effective cost-value ratio. Hence we made the decision early on to use ‘process’ rather than ‘analysis_process’. 

Hopefully this doesn’t cause any significant issues - but please let us know if this causes any unexpected burden to Azul!

Reply:
Happy to change that. It was more of an oversight on my part than a deliberate choice.

Reply:
Thank you!

Reply:
Addressed.